framework/web/Session.php

```diff
@@ -129,6 +131,7 @@ public function open()
         if ($this->getIsActive()) {
             Yii::info('Session started', __METHOD__);
             $this->updateFlashCounters();
+            register_shutdown_function([$this, 'close']);
```

Comment:
Ah, there is no unregister function in PHP, got it.
Should I return original code? Or use some kind of flag to check handler registration?